analysers/analyser_merge_power_tower_FR.py

```diff
@@ -45,15 +45,16 @@ def __init__(self, config, logger = None):
             Conflate(
                 select = Select(
                     types = ["nodes"],
-                    tags = [{"power": "tower", "operator": "RTE"}, {"power": "pole", "operator": "RTE"}, {"power": "terminal", "operator": "RTE"}, {"power": "portal", "operator": "RTE"}, {"power": "insulator", "operator": "RTE"},
-                      {"power": "tower", "operator": False}, {"power": "pole", "operator": False}, {"power": "terminal", "operator": False}, {"power": "portal", "operator": False}, {"power": "insulator", "operator": False}]),
+                    tags = [{"power": ["tower", "pole", "terminal", "portal", "insulator"], "operator":[False, "RTE"]}]),
 #                osmRef = "ref:FR:RTE", # Commented initial. Only issues missing tower. Then when the missing tower number lower, uncomment to integrate ref into OSM.
                 conflationDistance = 10,
                 mapping = Mapping(
                     static1 = {
-                        "power": "tower",
-                        "operator": "RTE"},
-                    static2 = {"source": self.source},
+                        "operator": "RTE",
+                        "operator:wikidata": "Q2178795"},
```

Comment:
If you set it in static1 it will raise again an osmose issue for each existing power "tower" to add `operator:wikidata`.

Reply:
Still 128k miss wikidata so I put it in static2